src/index.js

```diff
@@ -0,0 +1,159 @@
+"use strict";
+
+//to keep track of and to be able to modify current state
+const state = {
+    city: 'SanFrancisco',
+    temp: 60,
+    latitude: null,
+    longitude: null
+}
+
+// would love to not need this and change the units in params instead
+const convertKtoF = (temp) => {
+    return (temp - 273.15) * (9 / 5) + 32;
+  };
```

Comment:
What do you mean by changing the units in the params? I think this approach of putting the logic into a helper method works well!